apps/build-onchain-apps/contracts/README.md

```diff
@@ -0,0 +1,75 @@
+## Contracts
+
+This project is built using Foundry. For more information, visit the docs [here](https://book.getfoundry.sh/)
+
+## Introduction
+
+This repository contains a sample `BuyMeACoffee.sol` contract which allows the user to buy the owner a coffee with `0.001 ether`. Along with that the user can send the owner a memo.
+
+## Project Layout
+
+```
+
+.
+├── foundry.toml
+├── script
+│   └── BuyMeACoffee.s.sol
+├── src
+│   └── BuyMeACoffee.sol
+└── test
+    └── BuyMeACoffee.t.sol
+
+```
+
+- You can configure Foundry's behavior using foundry.toml.
+- The default directory for contracts is src/.
+- The default directory for tests is test/
+- The default directory for writing scripts is script/
+
+## Usage
+
+### Installation
+
+Install foundry using
+
+```shell
+curl -L https://foundry.paradigm.xyz | bash
+foundryup
+```
+
+### Build
+
+```shell
+$ forge build
+```
+
+### Test
+
+```shell
+$ forge test
+```
+
+### Format
+
+```shell
+$ forge fmt
+```
+
+### Deploy to Base Goerli
+
+Create a `.env` file using the `.env.example` file provided.
+
+```shell
+$ source .env
+$ forge script script/BuyMeACoffee.s.sol:BuyMeACoffeeScript --broadcast --verify --rpc-url https://goerli.base.org --etherscan-api-key ${BASESCAN_API_KEY}
```

Comment:
@Sneh1999 Ah I even would have just hardcoded it in the toml file

Reply:
Yeah so maybe just `RPC_URL` and also `BLOCK_EXPLORER_URL` so that we don't make the variable names network-specific? Although I guess then we run into having to make the object keys generic as well

Reply:
I will update the names for now. I think this is a starting point but yes, as we progress we can make things more general.